packages/app/src/components/transactions/Status.vue

```diff
@@ -241,6 +290,57 @@ const remainingTxStatuses: RemainingStatus[] = [
   },
 ];
 
+// Helper function to check if we should show Ethereum status
+const shouldShowEthereumStatus = (): boolean => {
+  // Only show Ethereum badge if this transaction actually used Gateway as settlement
+  return isGatewayTransaction();
+};
+
+// Helper function to check if transaction uses Gateway settlement
+const isGatewayTransaction = (): boolean => {
+  // If chain IDs are available, check them directly
+  if (props.commitChainId || props.proveChainId || props.executeChainId) {
```

Comment:
I think `if (props.commitChainId || props.proveChainId || props.executeChainId)` can be simplified to `if (props.commitChainId)`. It's the first chain id that will be available.